lesson_06/homeworks_06.py

```diff
@@ -4,13 +4,29 @@
 Напишіть оператор if, щоб перевірити, чи колір прибульця 'green'.
 Якщо так, надрукуйте повідомлення про те, що гравець щойно заробив 5 балів.
 """
+alien_color = "red"
+colour_green = "green"
+colour_red = "red"
+colour_yellow = "yellow"
+
+if alien_color in colour_green:
+    print('Ви тільки що заробили 5 балів')
 
 # task 2
 """  Скопіюйте пеопередню відповідь, змініть і доповніть її умовою else.
 Якщо колір прибульця зелений, надрукуйте, що гравець щойно заробив 5 балів.
 Якщо колір прибульця не зелений, надрукуйте, що гравець щойно заробив 10 балів.
 Зробіть так, щоб виводилася умова else.
 """
+alien_color = "red"
+colour_green = "green"
+colour_red = "red"
+colour_yellow = "yellow"
```

Comment:
```suggestion
alien_color = "red"
colour_green = "green"
colour_red = "red"
colour_yellow = "yellow"
```
оце неправильно, ми просто використовуємо змінні оголошені рашіше

Reply:
Не дуже зрозумів, я наче і не змінював змінні і в 1му та другому завданні працював з ними, чи що мається на увазі?

Reply:
Зрозумів, коли Ви ставите "-" це рядки котрі потрібно прибрати